src/app/project/project.component.spec.ts

```diff
@@ -3,38 +3,18 @@ import { RouterTestingModule } from '@angular/router/testing';
 import { ActivatedRoute } from '@angular/router';
 import { MomentModule } from 'angular2-moment/moment.module';
 
+import { ProjectFactory } from '../tests/project-factory';
+import { BuildFactory } from '../tests/build-factory';
+
 import { ProjectComponent } from './project.component';
+import { Project } from '../models/Project';
+import { Build } from '../models/Build';
 
 describe('ProjectComponent', () => {
   let component: ProjectComponent;
   let fixture: ComponentFixture<ProjectComponent>;
   let project;
-
-  project = {
-    id: 'brigade-29d38c7477ecee18e184b69bec354fc350605c51bc16d4dd2b6073',
-    name: 'kashti/localdev',
-    repo: {
-      name: 'github.com/kashti/localdev',
-      cloneURL: 'https://github.com/kashti/localdev.git'
-    },
-    defaultScript: '',
-    kubernetes: {
-      namespace: 'default',
-      vcsSidecar: '',
-      buildStorageSize: '50Mi'
-    },
-    github: {
-      baseURL: '',
-      uploadURL: ''
-    },
-    secrets: {},
-    worker: {
-      registry: '',
-      name: '',
-      tag: '',
-      pullPolicy: ''
-    }
-  };
+  let builds;
```

Comment:
Can we also have the test strongly-typed?
(Not a blocker for this PR though)

Reply:
Done, thanks for catching.